worlds/messenger/docs/setup_en.md

```diff
@@ -1,31 +1,29 @@
 # The Messenger Randomizer Setup Guide
 
 ## Quick Links
-- [Main Page](../../../../games/The%20Messenger/info/en)
+- [Game Info](../../../../games/The%20Messenger/info/en)
 - [Settings Page](../../../../games/The%20Messenger/player-settings)
 - [Courier Github](https://github.com/Brokemia/Courier)
 - [The Messenger Randomizer Github](https://github.com/minous27/TheMessengerRandomizerMod)
 - [Jacksonbird8237's Item Tracker](https://github.com/Jacksonbird8237/TheMessengerItemTracker)
 - [PopTracker Pack](https://github.com/alwaysintreble/TheMessengerTrackPack)
 
-## Required Software
-
-- [The Messenger](https://store.steampowered.com/app/764790/The_Messenger/)
-  - Only Steam version is currently supported.
-- [Courier Mod Loader](https://github.com/Brokemia/Courier/releases)
-- [The Messenger Randomizer Mod](https://github.com/minous27/TheMessengerRandomizerMod/releases)
-
 ## Installation
 
 1. Download and install Courier Mod Loader using the instructions on the release page
+   * [Latest release is currently 0.7.1](https://github.com/Brokemia/Courier/releases)
 2. Download and install the randomizer mod
-     * Download the latest `TheMessengerRandomizer.zip`
-     * Extract the zip file to `TheMessenger/Mods/` of your game's install location
-     * Optionally, Backup your save game
+   1. Download the latest [TheMessengerRandomizer.zip](https://github.com/minous27/TheMessengerRandomizerMod/releases)
```

Comment:
This matches more closely what we have elsewhere. (I don't have a strong opinion). Above (Courier) has a similar situation
```suggestion
   1. Download the latest TheMessengerRandomizer.zip from the [The Messenger Randomizer Mod releases page](https://github.com/minous27/TheMessengerRandomizerMod/releases)
```